src/scenes/Event/store.js

```diff
@@ -54,6 +57,27 @@ export default class {
   @observable buttonDisabled = INIT.buttonDisabled
   @observable warningType = INIT.warningType
   @observable eventWarningMessageId = INIT.eventWarningMessageId
+  // topic
+  @observable topics = []
+  withdrawableAddresses = []
+  betBalances = []
+  voteBalances = []
+  @computed get totalBetAmount() {
+    return _.sum(this.betBalances);
+  }
+  @computed get totalVoteAmount() {
+    return _.sum(this.voteBalances);
+  }
+  @computed get resultBetAmount() {
+    return this.betBalances[this.selectedOptionIdx];
+  }
+  @computed get resultVoteAmount() {
+    return this.voteBalances[this.selectedOptionIdx];
+  }
+  @computed get topic() {
+    return _.find(this.topics, { address: this.address }) || {};
```

Comment:
can we just make this a single instance instead of an array? there should only be 1 topic per event ever.

Reply:
@henryzbw

Reply:
`.find` already returns an object

Reply:
I don't know what you mean, the return value here is an object.
I console.log `typeof` shows it's an object
![image](https://user-images.githubusercontent.com/10339131/43805248-ea058c4c-9a53-11e8-8c8b-cac1a63ae1e6.png)

Reply:
ya you are right. sorry about that. was reviewing this at like 2am.

Reply:
thank @dwalintukan, take care, it's flu season.